SidesTriangle.java

```diff
@@ -0,0 +1,21 @@
+import java.util.Scanner;
+
+public class SidesTriangle {
+    public static void main(String[] args) {
+        Scanner scanner = new Scanner(System.in);
+
+        System.out.print("Enter  first side: ");
+        int a = scanner.nextInt();
+        System.out.print("Enter  second side: ");
+        int b = scanner.nextInt();
+        System.out.print("Enter third side: ");
+        int c = scanner.nextInt();
+
+
+        if ((a + b > c) && (a + c > b) && (b + c > a)) {
```

Comment:
Condition is correct, but please think and check for any kind of inputs and update the code.
Your code lacks for inputs validation.